### PR TITLE
Sample Design Of An Mobile Ecommerce App (BuyMob) Is Added

### PR DESCRIPTION
**Pull request by nijin26**
## Description

This is a design of an eCommerce app where we can buy mobiles. It has designs of pages like Authentication, Home, Orders, Carts, Payment etc.

## Related Issue

None

## Type of Change:

- [x] Ui/Ux
- [ ] Coding
- [ ] Documentation
- [ ] Other : 

## Checklist:

- [x] Pull request is made against `gssoc` branch.
- [x] My code/design follows the style guidelines of this project.
- [x] I have performed a self-review of my own code/design.
- [ ] I have commented my code, particularly in hard-to-understand areas(or gave a proper description about the design).
- [ ] My UI is responsive
- [x] UI made is my unique.

### CODE REVIEW

**Comment by SHITIZ-AGGARWAL** [comment]
@nijin26 the folder name and the name of html file should be based on the ui design

**Comment by nijin26** [comment]
@SHITIZ-AGGARWAL oh sorry. I got confused between instructions on Contribution.md & HTML Template given.
I will change it.

**Comment by nijin26** [comment]
@SHITIZ-AGGARWAL Fixed 🙂

File: Designs/BuyMob/BuyMob.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl"
+      crossorigin="anonymous"
+    />
+
+    <title>Your Name</title>
```
Comment (SHITIZ-AGGARWAL) [inline Designs/BuyMob/BuyMob.html]: add you name in it

**Review by SHITIZ-AGGARWAL — CHANGES_REQUESTED** [review]

File: Designs/BuyMob/BuyMob.html (hunk 1 above)
Comment (nijin26) [reply Designs/BuyMob/BuyMob.html]: Done !

**Comment by awantika10** [comment]
@nijin26 You have designed an application which is very common

**Comment by awantika10** [comment]
@nijin26 Closing as you need to first make an issue for the design

**Comment by SHITIZ-AGGARWAL** [comment]
@nijin26 For designing PRs , only the ones which are solving an issue will be counted. If you want to suggest a design , you can talk about it in the #issue-discussion or #design-discussion in the discord channel and if we approve it then you can raise an issue and then link a pr to it.

**Pull request closed without merging**